src/api/store/accounts.sagas.test.ts

```diff
@@ -1,33 +1,17 @@
-import type { TAddress } from '@mycrypto/wallets';
-import { DEFAULT_ETH } from '@mycrypto/wallets';
+// @fixme DONT DO THIS
```

Comment:
What´s the reason for this?

Reply:
We probably want to move this to the crypto process since it touches private keys. Just haven't moved it yet